src/components/SelectionList/SearchTableHeader.tsx

```diff
@@ -85,6 +86,18 @@ const SearchColumns: SearchColumnConfig[] = [
     },
 ];
 
+function getSearchColumns(type: SearchDataTypes): SearchColumnConfig[] {
+    switch (type) {
+        case CONST.SEARCH.DATA_TYPES.TRANSACTION:
+        case CONST.SEARCH.DATA_TYPES.REPORT:
+        case CONST.SEARCH.DATA_TYPES.EXPENSE:
+        case CONST.SEARCH.DATA_TYPES.INVOICE:
+            return expenseHeaders;
+        default:
+            return expenseHeaders;
+    }
+}
```

Comment:
I'm confused by this function, doesn't it just return `expenseHeaders` everytime? Why have the switch statement?

Reply:
Yes, for now that's all it does. We'll start returning other headers soon.